seedDB/README.md

```diff
@@ -5,6 +5,25 @@ These files are MySQL database dump files for seeding a new instance of the cBio
 The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database schema 2.1.0. This means that the version numbers won't be identical. cBioPortal 1.9.0 with database schema 2.4.0 removed PDB annotations from the database.
 
 ## Latest seed database
+#### Seed database schema 2.8.2
+
+This schema is required for cBioPortal release versions:
+- **2.8.2**
+
+When using a release version **> 2.0.0**, a migration step to a new database schema might be required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
+
+**Schema 2.8.2**: [SQL file with create table statements](https://raw.githubusercontent.com/cBioPortal/cbioportal/451cdeacb06606369ff13b64219fce70388c679b/db-scripts/src/main/resources/cgds.sql)<br>
```

Comment:
ok, this is clear ... here we are telling the user that they should use these instructions for installing to schema 2.8.2